### PR TITLE
Add content of Privacy Policy (with Google Docs link)

### LINKED ISSUE #96: Add Privacy Policy

### PR DESCRIPTION
**Pull request by alexisreytech**
@brian-montgomery, I used the html strong element here in one spot. Is this usage okay, or should it be changed to just use the CSS class of "fw-bold"?

Fixes #96

### CODE REVIEW

File: astro/src/pages/privacy-policy.astro [hunk 1]
```diff
@@ -5,5 +5,115 @@ import ThemedSection from '../components/ThemedSection.astro';
 
 <Layout title="Privacy Policy">
  <ThemedSection style="secondary">
+  <p>Last updated: <time datetime="2023-01-15">15 January 2023</time></p>
```
Comment (brian-montgomery) [inline astro/src/pages/privacy-policy.astro:8]: It's 2024. :)
Perhaps right-align this?

**Comment by alexisreytech** [comment]
@brian-montgomery, decided to just go with CSS for now, so it more closely matches what's on Google Docs. Should be good.

**Review by brian-montgomery — CHANGES_REQUESTED** [review]
Perhaps add some vertical margin or padding to the `h2` and `h3` tags here, to space it out a bit? The document feels really cramped compared to the rest of the site.

File: astro/src/pages/privacy-policy.astro (hunk 1 above)
Comment (alexisreytech) [reply astro/src/pages/privacy-policy.astro:8]: @brian-montgomery, that was the date on the document I got, so I just copied it over.

**Comment by alexisreytech** [comment]
> Perhaps add some vertical margin or padding to the `h2` and `h3` tags here, to space it out a bit? The document feels really cramped compared to the rest of the site.

@brian-montgomery, I understand what you're saying here. The plain language summary uses more list formatting, so there is less space in this document. I just tried to match how the other policies looked.

**Review by brian-montgomery — APPROVED** [review]

**Pull request merged**